homeassistant/components/microbees/climate.py

```diff
@@ -0,0 +1,142 @@
+"""Climate integration microBees."""
+
+from typing import Any
+
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import MicroBeesUpdateCoordinator
+from .entity import MicroBeesActuatorEntity
+
+CLIMATE_PRODUCT_IDS = {76: "Thermostat", 78: "Thermovalve"}
```

Comment:
This is fine, but considering we don't use the values of the dict I prefer this:
```suggestion
CLIMATE_PRODUCT_IDS = {
    76,  # Thermostat,
    78,  # Thermovalve,
}
```